core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -685,7 +685,8 @@ class MasterSuite extends SparkFunSuite
     }
   }
 
-  test("SPARK-27510: Master should avoid dead loop while launching executor failed in Worker") {
+  // TODO(SPARK-32250): Enable the test back. It is flaky in GitHub Actions.
+  ignore("SPARK-27510: Master should avoid dead loop while launching executor failed in Worker") {
```

Comment:
cc @Ngone51

Reply:
Got it. It has been tracked by SPARK-32250